spec/rspec/support/encoded_string_spec.rb

```diff
@@ -209,9 +246,9 @@ module RSpec::Support
 
           it 'makes no changes to the resulting string' do
             resulting_array = build_encoded_string(non_ascii_compatible_string).split("\n")
-            expect(resulting_array.size).to eq(1) # sanity check
-            expected_string = non_ascii_compatible_string
-            expect_identical_string(resulting_array.first, expected_string, Encoding.find('UTF-16LE'))
+            expect(resulting_array).to match [
```

Comment:
`match` is a matcher that supports matching array/hash data structures using embedded matchers to match against items in the actual array or hash. See:

http://myronmars.to/n/dev-blog/2014/05/notable-changes-in-rspec-3#_matcher_can_be_used_for_data_structures

In this case, it allows us to assert that `result_array` is an array of one element, which matches the `a_string_identical_to` matcher (defined as an alias of `be_identical_string` above), all in a single expression -- that way, you don't have to do the separate `expect(resulting_array.size).to eq(1)` sanity check.